### PR TITLE
add image lazy load

### PR DESCRIPTION
**Pull request by dungphanxuan**
todo #69

### CODE REVIEW

File: assets/LazyLoadAsset.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+/**
+ * CodeMirrorAsset groups assets for code editing areas
+ */
+class LazyLoadAsset extends AssetBundle
+{
+    public $sourcePath = null;
+
+    public $js = [
+        'https://cdnjs.cloudflare.com/ajax/libs/vanilla-lazyload/10.5.2/lazyload.min.js',
```
Comment (samdark) [inline assets/LazyLoadAsset.php]: I'd load it from local server instead.

File: assets/LazyLoadAsset.php [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<?php
+
+
+namespace app\assets;
+
+
+use yii\web\AssetBundle;
+
+/**
+ * CodeMirrorAsset groups assets for code editing areas
```
Comment (samdark) [inline assets/LazyLoadAsset.php]: It's not CodeMirrorAsset

File: assets/LazyLoadAsset.php [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+ */
+class LazyLoadAsset extends AssetBundle
+{
+    public $sourcePath = null;
```
Comment (samdark) [inline assets/LazyLoadAsset.php]: No need for `= null`, it's default.

**Comment by dungphanxuan** [comment]
@samdark , i've been updated about your comment

**Comment by samdark** [comment]
Merged and deployed. Thank you!

**Pull request closed without merging**